testutil/timeouts.go

```diff
@@ -20,7 +20,7 @@
 package testutil
 
 import (
-	"runtime"
+	"github.com/snapcore/snapd/arch"
```

Comment:
*util packages should not use non *util packages

Reply:
🧐🤔😳

For now we have 1:1 mapping between goarch and snapd arches, so either is fine. It felt to me that using arch would be more natural.

I guess this function could have been moved to the arch package directly. If and when we might want host arch dependant timeouts at normal runtime as well.